modules/stochastic_tools/test/tests/multiapps/sampler_multiapp/master_full_solve.i

```diff
@@ -0,0 +1,42 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+[]
+
+[Problem]
+  kernel_coverage_check = false
+  solve = false
+[]
+
+[Distributions]
+  [./uniform_0]
+    type = UniformDistribution
+    lower_bound = 0.1
+    upper_bound = 0.3
+  [../]
+[]
+
+[Samplers]
+  [./mc]
+    type = MonteCarloSampler
+    n_samples = 5
+    distributions = 'uniform_0'
+  [../]
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+[MultiApps]
+  [./runner]
+    type = SamplerFullSolveMultiApp
+    sampler = mc
+    input_files = 'sub.i'
+#    positions = '0 0 0
```

Comment:
I'd prefer to just drop this parameter rather than comment it out.

Reply:
deleted!